.github/actions/file-diff/index.js

```diff
@@ -137,7 +140,7 @@ async function run() {
 							...(hasDiff
 								? [
 									bytesToSize(totalDiffSize),
-									`${printChange(totalDiffSize - totalSize)} (${printPercentChange((totalDiffSize - totalSize) / totalSize)})`,
+									`${printChange(totalDiffSize - totalSize)}${totalDiffSize - totalSize !== 0 ? ` (${printPercentChange((totalDiffSize - totalSize) / totalSize)})` : ""}`,
```

Comment:
We only want to report on a percent changed if the file size has actually changed :)